Selu383.SP24.Web/package.json

```diff
@@ -1,39 +1,42 @@
 {
-  "name": "selu383-sp24-web",
-  "private": true,
-  "version": "0.0.0",
-  "type": "module",
+  "name": "selu.383.sp24.mobile",
```

Comment:
These changes are likely undesired - I suggest you revert them.
This is for the web - not expo